Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlayPointer.java

```diff
@@ -41,8 +44,34 @@ public InputOverlayPointer(Context context, int button)
     DisplayMetrics outMetrics = new DisplayMetrics();
     display.getMetrics(outMetrics);
     doubleTapButton = button;
-    maxWidth = outMetrics.widthPixels / 2;
-    maxHeight = outMetrics.heightPixels / 2;
+
+    // Adjusting for device aspect ratio. This only needs to be done on width since the render height
+    // will fill the screen
```

Comment:
Is this really correct? What about portrait mode, for instance?

Reply:
Yeah the comment is old. I initially only had it setup to adjust for width for landscape play but it does now check where the black bars are and adjusts accordingly.